fenix/app/src/main/java/org/mozilla/fenix/components/Core.kt

```diff
@@ -281,6 +282,7 @@ class Core(
                 HistoryMetadataMiddleware(historyMetadataService),
                 SessionPrioritizationMiddleware(),
                 SaveToPDFMiddleware(context),
+                ExtensionsProcessMiddleware(context.components.core.engine),
```

Comment:
This seems like it should actually be in `EngineMiddleware.kt` [here](https://github.com/mozilla-mobile/firefox-android/blob/8bf8103b9c75c6d2cf876e9a2cbf59bac1cf9fb5/android-components/components/browser/state/src/main/java/mozilla/components/browser/state/engine/EngineMiddleware.kt#L50)